services/apps/premium/members_enrichment_worker/src/activities/enrichment.ts

```
@@ -136,3 +138,204 @@ export async function findMemberIdentityWithTheMostActivityInPlatform(
+      const normalized = (await normalizeEnrichmentData(
+        source,
+        cache.data,
+      )) as IMemberEnrichmentDataNormalized
+
+      // TODO uros temp remove contributions from sources to mitigate context size
+      if (Array.isArray(normalized)) {
+        const normalizedContributions = []
+        for (const n of normalized) {
+          if (n.contributions) {
+            normalizedContributions.push(n.contributions)
+            delete n.contributions
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)